src/ui/map.js

```diff
@@ -2640,6 +2636,10 @@ class Map extends Camera {
 
         if (this._loaded && !this._fullyLoaded && !somethingDirty) {
             this._fullyLoaded = true;
+            if (this._isInitialLoad) {
+                this._authenticate();
+            }
+            this._isInitialLoad = false;
```

Comment:
Moving this `_isInitialLoad` flip here means that fading will be active sooner than the first `idle` event. That may have an impact on load time.

Reply:
I moved `_isInitialLoad` back to the `idle` event and just call `_authenticate` here